internal/tiltfile/tiltfile_test.go

```diff
@@ -2657,6 +2697,17 @@ func cb(img imageHelper, opts ...interface{}) cbHelper {
 	return cbHelper{img, opts}
 }
 
+type nestedFBHelper struct {
```

Comment:
is there a reason why this is done differently than the custom build fb assertions? it seems weird for custom_build to do it one way and docker_build to do it another way.

Reply:
...shoot, good call